ui/packages/catalog-ui-search/src/main/webapp/component/map-context-menu/map-context-menu.view.js

```diff
@@ -27,74 +27,41 @@ var lightboxInstance = require('../lightbox/lightbox.view.instance.js')
 module.exports = Marionette.LayoutView.extend({
   template(props) {
     return (
-      <React.Fragment>
-        {typeof props.mouseLat === 'undefined' ||
-        typeof props.mouseLon === 'undefined' ? null : (
-          <CopyCoordinates
-            clickDms={props.clickDms}
-            clickLat={props.clickLat}
-            clickLon={props.clickLon}
-            clickMgrs={props.clickMgrs}
-            clickUtmUps={props.clickUtmUps}
-            closeParent={this.triggerClick.bind(this)}
-          />
-        )}
-        <div
-          className="metacard-interaction interaction-view-histogram"
-          data-help="Open histogram of results."
-        >
-          <div className="interaction-icon fa fa-bar-chart" />
-          <div className="interaction-text">View Histogram</div>
-        </div>
-        <div
-          className="metacard-interaction interaction-view-details"
-          data-help="Open inspector for result."
-        >
-          <div className="interaction-icon fa fa-info" />
-          <div className="interaction-text">
-            View Inspector
-            <div>
-              <span>{props.target}</span>
-            </div>
-          </div>
-        </div>
-        <div
-          className="metacard-interaction interaction-view-details-selection"
-          data-help="Open inspector for selected results."
-        >
-          <div className="interaction-icon fa fa-info" />
-          <div className="interaction-text">
-            View Inspector (Current Selection)
-            <div>
-              <span>{props.selectionCount} selected</span>
-            </div>
-          </div>
-        </div>
-        <div
-          className="metacard-interaction interaction-view-histogram-selection"
-          data-help="Open histogram of selected results."
-        >
-          <div className="interaction-icon fa fa-bar-chart" />
-          <div className="interaction-text">
-            View Histogram (Current Selection)
-            <div>
-              <span>{props.selectionCount} selected</span>
-            </div>
-          </div>
-        </div>
-      </React.Fragment>
+      <MapContextMenu
+        onChange={(value: string) => this.selectMenu(value)}
+        mouseLat={props.mouseLat}
+        mouseLon={props.mouseLon}
+        target={props.target}
+        clickDms={props.clickDms}
+        clickLat={props.clickLat}
+        clickLon={props.clickLon}
+        clickMgrs={props.clickMgrs}
+        clickUtmUps={props.clickUtmUps}
```

Comment:
✏️ There are 5 click options, I'd recommend changing them to a single prop called `clickValues` that has a key for dms, lat, lon, mgsr, and utmUps that way you can reduce the number of props you are sending.